content/pt-br/docs/reference/glossary/sysctl.md

```diff
@@ -0,0 +1,19 @@
+---
+title: sysctl
+id: sysctl
+date: 2019-02-12
+full_link: /docs/tasks/administer-cluster/sysctl-cluster/
+short_description: >
+  Uma interface para obter e definir parâmetros do kernel Unix.
+
+aka:
+tags:
+- tool
+---
+ `sysctl` é uma interface semi-padronizada para ler ou alterar os atributos do kernel Unix em execução.
+
+<!--more-->
+
+Em sistemas do tipo Unix-like, `sysctl` é tanto o nome da ferramenta que os administradores usam para visualizar e modificar essas configurações, quanto a chamada do sistema que a ferramenta usa.
```

Comment:
Suggestion: "sistemas do tipo Unix" reads more natural than "sistemas do tipo Unix-like" and delivers the same meaning.